doc(README): formatting, syntax updating, wording (#31)
apache/cordova-serve: README.md

#
-->

# cordova-serve

[![NPM](https://nodei.co/npm/cordova-serve.png)](https://nodei.co/npm/cordova-serve/)

[![Node CI](https://github.com/apache/cordova-serve/workflows/Node%20CI/badge.svg?branch=master)](https://github.com/apache/cordova-serve/actions?query=branch%3Amaster)

> This module can be used to serve up a Cordova application in the browser. It has no command-line, and is intended to be called APIs.

Reviewer: ```suggestion
This module provides a JavaScript API to serve up a Cordova application in the browser.
```
